src/Repeater/RepeaterLauncher.ts

```diff
@@ -3,8 +3,6 @@ export interface RepeaterLauncher {
 
   loadScripts(scripts: Record<string, string>): Promise<void>;
 
-  compileScripts(scripts: string | Record<string, string>): void;
```

Comment:
We are going to preserve this ability for the legacy implementation. Just mark as optional and deprecate:

```ts
  /**
   * @deprecated currently not supported by some implementations
   */
  compileScripts?(scripts: string | Record<string, string>): void;

```